.travis.yml

```diff
@@ -2,6 +2,11 @@ language: node_js
 
 sudo: false
 
+addons:
+  apt:
+    packages:
+    - time
```

Comment:
I thought this is built-in? I'm [able to run](https://travis-ci.com/curbengh/hexo-site/jobs/247537552) `/usr/bin/time` without adding this package. I'm using bionic in that test, perhaps it's not included in xenial?

Reply:
I had the same question. And I investigated. The ubuntu 16.04 server seems not installed `time` package by default.

> http://releases.ubuntu.com/16.04/
> http://releases.ubuntu.com/16.04/ubuntu-16.04.6-server-amd64.manifest